src/components/favorite/CardList/Card.tsx

```diff
@@ -9,33 +9,37 @@ interface CardProps {
     title: string;
     imageSource: string;
     description: string;
-    createAt: string;
+    createdAt: string;
```

Comment:
이 부분은 api에서 오는 card DTO인 것 같으니 타입을 따로 관리하면 다형성과 재사용성에서 좋을 것 같아요 😁
```suggestion
interface CardProps {
  card: FavoriteLinkTypes
}
```